src/example-types/results.js

```diff
@@ -28,22 +28,36 @@ module.exports = {
     if (context.include || context.exclude) result._source = {}
     if (context.include) result._source.includes = context.include
     if (context.exclude) result._source.excludes = context.exclude
+
     let highlight =
       _.getOr(true, 'highlight', context) && schema.elasticsearch.highlight
+    let inlineAliases = _.getOr(
+      {},
+      'elasticsearch.highlight.inlineAliases',
+      schema
+    )
+
     if (highlight) {
-      let schemaHiglightFields = _.flatten(
-        _.values(schema.elasticsearch.highlight)
-      )
-      let highlightFields = _.get('showOtherMatches', context)
```

Comment:
It looks like you killed support for `showOtherMatches`

Reply:
Added back the `showOtherMatches` support